tools/AutoTuner/src/autotuner/distributed.py

```diff
@@ -463,6 +469,15 @@ def parse_arguments():
     if args.timeout is not None:
         args.timeout = round(args.timeout * 3600)
 
+    # Validate cloud_dir if exist
+    if args.cloud_dir:
+        _ = CloudPath(args.cloud_dir)
+        if not args.cloud_dir.startswith("gs://"):
+            print(
+                f"[ERROR TUN-0030] Cloud storage directory {args.cloud_dir} is not supported."
+            )
+            sys.exit(1)
```

Comment:
Remove or transform into a warning. If changing to a warning, let the user know to look at the docs for more info.

Reply:
What happens if the path is invalid or the user did not specify it? If the goal is to have `CloudPath` fail, let's use a try/exception block to provide clear instructions on why it failed.